compose/material3/material3-adaptive-navigation-suite/build.gradle

```diff
@@ -63,8 +74,9 @@ androidXMultiplatform {
         androidMain {
             dependsOn(jvmMain)
             dependencies {
-                api("androidx.annotation:annotation:1.1.0")
-                api("androidx.annotation:annotation-experimental:1.4.0")
+                //dependencies are not available in JB fork
+                //api(project(":annotation:annotation"))
+                //api(project(":annotation:annotation-experimental"))
```

Comment:
Move the first one in the commonMain block and keep the second as a 1.4.0 binary